crates/types/src/rpc/transactions/broadcasted_invoke_transaction.rs

```diff
@@ -89,6 +89,7 @@ impl BroadcastedInvokeTransaction {
         Ok(InvokeTransaction {
             tx: starknet_api::transaction::InvokeTransaction::V1(sn_api_transaction),
             tx_hash: starknet_api::transaction::TransactionHash(txn_hash.into()),
+            only_query: false,
```

Comment:
`only_query` flag is related to estimate fee where the version of the txn is 2^128 + version. In this case a possible situation is to have `estimate_fee` request with `only_query` flag set to false.

Reply:
Thanks for the review, my fix is here: https://github.com/0xSpaceShard/starknet-devnet-rs/pull/241/commits/71b095a64befd8840738b1f0b61ac11b5f9a2c30
I'm passing only_query to be true in estimation and simulation, all other places are false.

Reply:
The problem is that I didn't know that this BroadcastedInvokeTransaction is used in estimation too, I thought it was used just in "real" transactions.